db/migrate/20160615205001_add_paid_at_to_auctions.rb

```diff
@@ -0,0 +1,24 @@
+class AddPaidAtToAuctions < ActiveRecord::Migration
+  def up
+    add_column :auctions, :paid_at, :datetime
+
+    execute <<-SQL
+      UPDATE auctions
+      SET paid_at = CURRENT_TIMESTAMP
```

Comment:
I know you have to do this because `paid_at` is also used like a boolean flag for payment. It would be good if we could retroactively reconcile this to have the correct times later, because it would skew things if we try to do a data analysis of average payment delays. Can you add an issue to fix `paid_at` in production once this PR is applied?